lbrynet/reflector/server/server.py

```diff
@@ -11,27 +11,89 @@
 
 log = logging.getLogger(__name__)
 
+SEND_SD_BLOB = 'send_sd_blob'
+SEND_BLOB = 'send_blob'
+RECEIVED_SD_BLOB = 'received_sd_blob'
+RECEIVED_BLOB = 'received_blob'
+NEEDED_BLOBS = 'needed_blobs'
+VERSION = 'version'
+BLOB_SIZE = 'blob_size'
+BLOB_HASH = 'blob_hash'
+SD_BLOB_SIZE = 'sd_blob_size'
+SD_BLOB_HASH = 'sd_blob_hash'
+
 
 class ReflectorServer(Protocol):
     def connectionMade(self):
         peer_info = self.transport.getPeer()
         log.debug('Connection made to %s', peer_info)
         self.peer = self.factory.peer_manager.get_peer(peer_info.host, peer_info.port)
         self.blob_manager = self.factory.blob_manager
+        self.protocol_version = self.factory.protocol_version
         self.received_handshake = False
-        self.received_descriptor = False
-        self.protocol_version = REFLECTOR_V2
         self.peer_version = None
         self.receiving_blob = False
         self.incoming_blob = None
         self.blob_write = None
-        self.blob_finished_d = None
+        self.blob_f_d = None
```

Comment:
`f` usually stands for file not finished; generally fail on the side of being verbose.